### ISSUE TITLE
[Phase 3] Self-play RL via single-token DPO (gated on Phase 2)

### ISSUE THREAD

**Issue by fritzprix** [fritzprix/systemone-lite#1]
## Status
**Design ready. Blocked on Phase 2 exit gate** ([#2](https://github.com/fritzprix/systemone-lite/issues/2)).

Do **not** run large self-play DPO loops on a policy that still fails bare-label spatial / weak chess (~0.24 move acc). VRAM enablers live in [#3](https://github.com/fritzprix/systemone-lite/issues/3).

Roadmap: [`docs/ROADMAP.md`](https://github.com/fritzprix/systemone-lite/blob/main/docs/ROADMAP.md)

## Motivation
Phase 1/2 SFT is behavioral cloning. Self-play DPO can optimize from win/loss once a competent enough reference policy exists.

System One scores **option aliases** (single-token decision), so DPO reduces to log-ratio over two scalar option logits — cheap vs sequence DPO.

## Pipeline
```text
[ Phase 2 spatial-v2 checkpoint = π_ref ]
        │
        ▼
[ Self-play rollouts ]  (chess first; Connect4 later)
        │
        ▼
[ Preference pairs ]  (chosen vs rejected / blunder pairs)
        │
        ▼
[ Single-token DPO / GRPO ]  (+ offline ref logprob cache from #3)
        │
        ▼
[ Eval ]  self-play score / Elo-style vs SFT + simple baselines
```

## Checklist
- [ ] Confirm Phase 2 exit gate passed (link report JSON in a comment)
- [ ] `scripts/chess_self_play.py` — model vs model rollouts, temperature / ε-greedy
- [ ] `scripts/build_chess_dpo.py` — outcome + blunder preference JSONL
- [ ] `scripts/chess_dpo.py` — single-token DPO using cached ref logprobs (#3)
- [ ] Tournament / Elo-style eval vs SFT and capped Stockfish / minimax
- [ ] Document latency on a **declared short payload** (do not cite multi-question demo averages as “sub-15ms”)

## Notes
- Baseline π_ref: prefer `checkpoints/systemone-spatial-v2` once available; else Phase 1 mixed only for tiny smoke tests.
- Hardware: RTX 3060 12GB with #3 memory stack.
